examples/ASDKgram/Sample/PhotoCellNode.m

```diff
@@ -29,6 +30,30 @@
 #define InsetForHeader UIEdgeInsetsMake(0, HORIZONTAL_BUFFER, 0, HORIZONTAL_BUFFER)
 #define InsetForFooter UIEdgeInsetsMake(VERTICAL_BUFFER, HORIZONTAL_BUFFER, VERTICAL_BUFFER, HORIZONTAL_BUFFER)
 
+
+// If an ASTextNode2 is member of a node tree that contains as accessibility container
+// it will be represented as a UIAccessibilityCustomAction attached to the accessibility container node. The UIAccessibilityCustomAction has as target the
+// ASTextNode2 itself as well as 
+@interface ASTextNode2(UIAccessibilityCustomAction)
+
+- (BOOL)performAccessibilityCustomAction:(UIAccessibilityCustomAction *)action;
```

Comment:
Should we wire this into the existing `textNode:tappedLinkAttribute:value:atPoint:textRange` delegate call?

Reply:
Convert frame.center to text node from accessibilityFrame?

Reply:
Done